keyboards/preonic/keymaps/arkag/keymap.c

```diff
@@ -0,0 +1,79 @@
+    KC_GRV,  KC_LPRN, KC_RPRN, KC_PLUS, KC_UNDS, KC_CIRC, KC_AMPR, KC_7,    KC_8,    KC_9,    _______, _______,
+    KC_LCBR, KC_LBRC, KC_RBRC, KC_EQL,  KC_MINS, KC_ASTR, KC_PIPE, KC_4,    KC_5,    KC_6,    _______, KC_NLCK,
+    KC_RCBR, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_PDOT, KC_1,    KC_2,    KC_3,    KC_BSLS, KC_PENT,
+    M_DASH,  KC_TILD, _______, KEEB,    _______, _______, _______, KC_0,    KC_PSLS, KC_PMNS, KC_PPLS, KC_PAST),
+
+  [_MEDIA] = LAYOUT_preonic_grid(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, KC_VOLU, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, KC_MPRV, KC_VOLD, KC_MNXT, _______, _______, _______, _______, _______, _______, _______,
+    KC_MPLY, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    KC_MUTE, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______),
+
+  [_ARROW] = LAYOUT_preonic_grid(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, KC_UP,   _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______),
+
+  [_FUNCT] = LAYOUT_preonic_grid(
+    KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_CAPS,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, M_OS   ),
+
+  [_KEEB] = LAYOUT_preonic_grid(
+    CK_TOGG, AU_TOG,  MU_TOG,  _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    M_USSR,  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, BL_TOGG, BL_STEP, BL_INC,  BL_DEC,  BL_BRTG, _______, _______, _______, _______, _______, _______,
+    RGB_M_P, RGB_TOG, RGB_MOD, RGB_VAI, RGB_VAD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, _______, _______, _______,
+    RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______),
+
+};
+
+#ifdef AUDIO_ENABLE
+    bool music_mask_user(uint16_t keycode) {
```

Comment:
```suggestion
bool music_mask_user(uint16_t keycode) {
```

Reply:
I don't understand this right now, with the changes I just pushed, si this resolved?

Reply:
It's a formatting/indentation change, that's all.  Nothing major.